platform/security/filter/security-filter-authorization/src/main/java/org/codice/ddf/security/filter/authorization/AuthorizationFilter.java

```diff
@@ -80,7 +80,9 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 
             boolean permitted = true;
 
-            final String path = httpRequest.getRequestURI();
+            final String path = httpRequest.getParameter("prevurl") == null ?
```

Comment:
We can't do this. This would allow anyone to hit any context by providing a query param in the URL to point towards something they have permission to view.